packages/grafana-data/tsconfig.json

```diff
@@ -1,19 +1,11 @@
 {
-  "extends": "../../tsconfig.json",
+  "extends": "../tsconfig.json",
   "include": ["src/**/*.ts", "src/**/*.tsx", "../../public/app/types/jquery/*.ts"],
   "exclude": ["dist", "node_modules"],
   "compilerOptions": {
     "rootDirs": ["."],
-    "module": "esnext",
-    "outDir": "compiled",
-    "declaration": true,
-    "declarationDir": "dist",
-    "strict": true,
-    "alwaysStrict": true,
-    "noImplicitAny": true,
-    "strictNullChecks": true,
     "typeRoots": ["./node_modules/@types", "types"],
-    "skipLibCheck": true, // Temp workaround for Duplicate identifier tsc errors,
-    "removeComments": false
+    "declarationDir": "dist",
```

Comment:
Can't this be shared? I assume it makes the `dist` being relative to the shared tsconfig.json location, righht?

Reply:
>All relative paths found in the configuration file will be resolved relative to the configuration file they originated in.

So yeah, that's the case. One solution might be to write `**/dist`?

Reply:
I'm not sure this would work. Ok let's leave it as it is now.